test/integration_tests/source_separation_pipeline_test.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+import torch
+import torchaudio
+from torchaudio.prototype.pipelines import CONVTASNET_BASE_LIBRI2MIX
+
+
+@pytest.mark.parametrize(
+    "bundle,task",
+    [
+        [CONVTASNET_BASE_LIBRI2MIX, "speech_separation"],
+    ],
+)
+def test_source_separation_models(bundle, task, mixture_speech, expected_tensor):
+    """Smoke test of source separation pipeline"""
+    separator = bundle.get_separator()
+    mixture_speech, sample_rate = torchaudio.load(mixture_speech)
+    expected_tensor = torch.load(expected_tensor)
+    mixture_speech = mixture_speech.reshape(1, 1, -1)
+    estimated_sources = separator(mixture_speech)
```

Comment:
would it be sufficient/better to test that `expected_tensor` and `estimated_sources` are equal?

Reply:
@carolineechen It's not the randomness that is an issue, but the fact that future minor changes in PyTorch would slightly change the result from time to time.

Using reference from a past run and comparing against it is not maintainable. It will suddenly break due to some change in core that happens in deep down of some BLAS routine change.

Reply:
hmm actually when I compared the waveforms on CI, the current test already failed, but I guess the error is tolerable if we set up a threshold the test can pass.